{"uuid": "1c2f048a-3397-4539-8a32-e1767248fc46", "children": ["dc7ce7be-83ba-40c0-a37d-baa7034cfbd4"], "befores": [{"name": "remote_driver", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"d:\\工具\\python\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"d:\\工具\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"d:\\工具\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\selenium_compass\\cases\\test_case_pytest.py\", line 28, in remote_driver\n    driver = webdriver.Remote('http://127.0.0.1:4444/wd/hub',desired_caps)\n  File \"d:\\工具\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 275, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"d:\\工具\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 365, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"d:\\工具\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 428, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"d:\\工具\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 347, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"d:\\工具\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 369, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"d:\\工具\\python\\lib\\site-packages\\urllib3\\request.py\", line 78, in request\n    return self.request_encode_body(\n  File \"d:\\工具\\python\\lib\\site-packages\\urllib3\\request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"d:\\工具\\python\\lib\\site-packages\\urllib3\\poolmanager.py\", line 375, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"d:\\工具\\python\\lib\\site-packages\\urllib3\\connectionpool.py\", line 699, in urlopen\n    httplib_response = self._make_request(\n  File \"d:\\工具\\python\\lib\\site-packages\\urllib3\\connectionpool.py\", line 445, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"d:\\工具\\python\\lib\\site-packages\\urllib3\\connectionpool.py\", line 440, in _make_request\n    httplib_response = conn.getresponse()\n  File \"d:\\工具\\python\\lib\\http\\client.py\", line 1344, in getresponse\n    response.begin()\n  File \"d:\\工具\\python\\lib\\http\\client.py\", line 307, in begin\n    version, status, reason = self._read_status()\n  File \"d:\\工具\\python\\lib\\http\\client.py\", line 268, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"d:\\工具\\python\\lib\\socket.py\", line 669, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1657609799709, "stop": 1657610069384}], "start": 1657609799709, "stop": 1657610071234}